### DISCUSSION TITLE
flexible http request path in online/offline mode

### DISCUSSION THREAD

**Discussion by CyL0NG** [projectdiscovery/nuclei#4816] [Ideas]
when use template in offline(passive) mode，the request path must be "{{BaseURL}}" or "{{BaseURL}}/" or "/"，other path like "{{BaseURL}}/foo.html" will fail with "[FTL] Could not run nuclei: no templates provided for scan".

sometimes we need to use template both in online and offline mode, online mode has some custom paths to check,  in this situation, we have to write two http parts in template and their matchers and extractors are same. which is not convenient to maintain templates.

may be passive mode should not check request path, or support multi paths like bellow:
```
path:
  - "{{BaseURL}}"
  - "{{BaseURL}}/foo.html"
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
